mautic/mautic
# [UX] Add the same user account helper on the user settings page

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This update incorporates the user account helper into the user settings page, mirroring the functionality found in the user profile. The helper reminds users of the variable that auto-populates their name in signatures, ensuring consistency across the platform.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Users
3. Edit you
4. Look at the Signature field

![image](https://github.com/mautic/mautic/assets/116097999/76fce734-b886-4d84-965f-716c9ea8e396)
